src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1423,10 +1423,11 @@ void window_guest_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
 
     // Time in park
     screenCoords.y += LIST_ROW_HEIGHT + 1;
-    if (peep->TimeInPark != -1)
+    int32_t guestEntryTime = peep->GetParkEntryTime();
+    if (guestEntryTime != -1)
     {
         int32_t eax = gScenarioTicks;
```

Comment:
While you are at it, could you rename this variable to something more sensible such as ```timeInPark```

Reply:
Please do this in a follow-up PR, so we don't forget :D